### ISSUE TITLE
commit ecdcdd962c0c73fc65d6f390d43743157c81b4de break cursor word highlight

### ISSUE THREAD

**Issue by jemag** [nyngwang/murmur.lua#5]
Using latest version including commit ecdcdd962c0c73fc65d6f390d43743157c81b4de seems to break cursor word highlighting.

Works properly with b7fc2b36df33fca58efe3a9f5b487f1be4bf138e

Basic config used:

```lua
require("murmur").setup({
  max_len = 80, -- maximum word-length to highlight
  exclude_filetypes = {},
})
```

**Comment by nyngwang** [comment]
Yes, you're right! Let me fix this in 12 hours. (I was busy fixing my config due to some upstream TUI updates recently, sorry for my late reply)

**Comment by nyngwang** [comment]
@jemag I believe this is fixed by my ac4f407. Thanks for your reporting. And you can be the first one helping me to try this: `:hi murmur_cursor_rgb guibg=pink` and hopefully the cursor rgb can be changed dynamically :fire:.

**Comment by jemag** [comment]
Works great, thanks!
